internal/terminal/table_test.go

```diff
@@ -256,3 +260,29 @@ func TestTablePayload(t *testing.T) {
 		assert.Equal(t, data, payloadData[logFieldData])
 	})
 }
+
+func TestParseValue(t *testing.T) {
+	t.Run("parseValue should create reasonable string representations", func(t *testing.T) {
```

Comment:
can we convert this to a test table with descriptions for each?  something like:

```go
for _, tc := range []struct{

}{
  {"Should parse a nil value as blank string", nil, ""},
  // ...
}{
  t.Run(tc.description, func(t *testing.T){ ... })
}
```